wagtail/wagtailadmin/edit_handlers.py

```diff
@@ -340,6 +342,14 @@ def __init__(self, children, classname=""):
         self.classname = classname
 
     def bind_to_model(self, model):
+        col_count = " col" + str(int(math.floor(12 / len(self.children))))
+
+        # If child panel doesn't have a col# class then append default based on
+        # number of columns
+        for child in self.children:
+            if re.match("^.*?(col)[1-9]{1}(?![3-9])[0-2]{0,1}.*?$", child.classname) is None:
```

Comment:
I think this could be simplified to: `if not re.search(r'\bcol\d+\b'):`

(this is slightly broader in that it'll match any numeric `col#` classname rather than just 1-12, but I think that's probably a good thing)